Compilation errors while building Docker image

```
rizinorg/rizin#47 [linux/amd64 stage-0 15/15] RUN cmake -DCMAKE_PREFIX_PATH=/tmp/***-install/usr -DCMAKE_INSTALL_PREFIX=/usr -B build && cmake --build build && DESTDIR=/tmp/***-install cmake --build build --target install
rizinorg/rizin#47 158.8 [ 34%] Linking CXX static library libghidra_sleigh.a
rizinorg/rizin#47 159.0 [ 34%] Built target ghidra_sleigh
rizinorg/rizin#47 159.0 Scanning dependencies of target ghidra_libdecomp
rizinorg/rizin#47 159.0 [ 34%] Building CXX object ghidra/CMakeFiles/ghidra_libdecomp.dir/ghidra/Ghidra/Features/Decompiler/src/decompile/cpp/libdecomp.cc.o
rizinorg/rizin#47 161.5 [ 35%] Linking CXX static library libghidra_libdecomp.a
rizinorg/rizin#47 161.6 [ 35%] Built target ghidra_libdecomp
rizinorg/rizin#47 161.6 Scanning dependencies of target core_ghidra
rizinorg/rizin#47 161.7 [ 36%] Building CXX object CMakeFiles/core_ghidra.dir/src/core_ghidra.cpp.o
rizinorg/rizin#47 163.3 In file included from /tmp/***-install/usr/include/librz/rz_parse.h:10,
rizinorg/rizin#47 163.3                  from /tmp/***-install/usr/include/librz/rz_asm.h:12,
rizinorg/rizin#47 163.3                  from /tmp/***-install/usr/include/librz/rz_egg.h:4,
rizinorg/rizin#47 163.3                  from /tmp/***-install/usr/include/librz/rz_core.h:14,
rizinorg/rizin#47 163.3                  from /tmp/rz-ghidra/src/ArchMap.h:10,
rizinorg/rizin#47 163.3                  from /tmp/rz-ghidra/src/core_ghidra.cpp:9:
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:737:1: sorry, unimplemented: non-trivial designated initializers not supported
rizinorg/rizin#47 163.3  };
rizinorg/rizin#47 163.3  ^
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:737:1: sorry, unimplemented: non-trivial designated initializers not supported
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:737:1: sorry, unimplemented: non-trivial designated initializers not supported
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:742:1: sorry, unimplemented: non-trivial designated initializers not supported
rizinorg/rizin#47 163.3  };
rizinorg/rizin#47 163.3  ^
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:742:1: sorry, unimplemented: non-trivial designated initializers not supported
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h: In function 'RzAnalysisVarKind rz_analysis_var_kind_from_string(const char*)':
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:760:1: error: invalid conversion from 'int' to 'RzAnalysisVarKind' {aka 'rz_analysis_var_kind_t'} [-fpermissive]
rizinorg/rizin#47 163.3  RZ_ANALYSIS_AS_STRING_IMPL(RzAnalysisVarKind, var_kind, RzAnalysisVarKind_strings);
rizinorg/rizin#47 163.3  ^~~~~~~~~~~~~~~~~~~~~~~~~~
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:760:1: error: invalid conversion from 'int' to 'RzAnalysisVarKind' {aka 'rz_analysis_var_kind_t'} [-fpermissive]
rizinorg/rizin#47 163.3  RZ_ANALYSIS_AS_STRING_IMPL(RzAnalysisVarKind, var_kind, RzAnalysisVarKind_strings);
rizinorg/rizin#47 163.3  ^~~~~~~~~~~~~~~~~~~~~~~~~~
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h: In function 'RzAnalysisVarOriginKind rz_analysis_var_origin_kind_from_string(const char*)':
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:761:1: error: invalid conversion from 'int' to 'RzAnalysisVarOriginKind' [-fpermissive]
rizinorg/rizin#47 163.3  RZ_ANALYSIS_AS_STRING_IMPL(RzAnalysisVarOriginKind, var_origin_kind, RzAnalysisVarOriginKind_strings);
rizinorg/rizin#47 163.3  ^~~~~~~~~~~~~~~~~~~~~~~~~~
rizinorg/rizin#47 163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:761:1: error: invalid conversion from 'int' to 'RzAnalysisVarOriginKind' [-fpermissive]
rizinorg/rizin#47 163.3  RZ_ANALYSIS_AS_STRING_IMPL(RzAnalysisVarOriginKind, var_origin_kind, RzAnalysisVarOriginKind_strings);
rizinorg/rizin#47 163.3  ^~~~~~~~~~~~~~~~~~~~~~~~~~
rizinorg/rizin#47 164.7 make[2]: *** [CMakeFiles/core_ghidra.dir/build.make:63: CMakeFiles/core_ghidra.dir/src/core_ghidra.cpp.o] Error 1
rizinorg/rizin#47 164.7 make[1]: *** [CMakeFiles/Makefile2:119: CMakeFiles/core_ghidra.dir/all] Error 2
rizinorg/rizin#47 164.7 make: *** [Makefile:130: all] Error 2
rizinorg/rizin#47 ERROR: process "/bin/sh -c cmake -DCMAKE_PREFIX_PATH=/tmp/***-install/usr -DCMAKE_INSTALL_PREFIX=/usr -B build && cmake --build build && DESTDIR=/tmp/***-install cmake --build build --target install" did not complete successfully: exit code: 2

rizinorg/rizin#40 [linux/arm64 stage-0  8/15] RUN meson --prefix=/usr -Dinstall_sigdb=true /tmp/build && 	meson compile -C /tmp/build && 	meson install --destdir /tmp/***-install -C /tmp/build
rizinorg/rizin#40 CANCELED

rizinorg/rizin#54 [linux/386 stage-0 15/15] RUN cmake -DCMAKE_PREFIX_PATH=/tmp/***-install/usr -DCMAKE_INSTALL_PREFIX=/usr -B build && cmake --build build && DESTDIR=/tmp/***-install cmake --build build --target install
rizinorg/rizin#54 CANCELED
------
 > [linux/amd64 stage-0 15/15] RUN cmake -DCMAKE_PREFIX_PATH=/tmp/***-install/usr -DCMAKE_INSTALL_PREFIX=/usr -B build && cmake --build build && DESTDIR=/tmp/***-install cmake --build build --target install:
163.3 /tmp/***-install/usr/include/librz/rz_analysis.h: In function 'RzAnalysisVarOriginKind rz_analysis_var_origin_kind_from_string(const char*)':
163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:761:1: error: invalid conversion from 'int' to 'RzAnalysisVarOriginKind' [-fpermissive]
163.3  RZ_ANALYSIS_AS_STRING_IMPL(RzAnalysisVarOriginKind, var_origin_kind, RzAnalysisVarOriginKind_strings);
163.3  ^~~~~~~~~~~~~~~~~~~~~~~~~~
163.3 /tmp/***-install/usr/include/librz/rz_analysis.h:761:1: error: invalid conversion from 'int' to 'RzAnalysisVarOriginKind' [-fpermissive]
163.3  RZ_ANALYSIS_AS_STRING_IMPL(RzAnalysisVarOriginKind, var_origin_kind, RzAnalysisVarOriginKind_strings);
163.3  ^~~~~~~~~~~~~~~~~~~~~~~~~~
164.7 make[2]: *** [CMakeFiles/core_ghidra.dir/build.make:63: CMakeFiles/core_ghidra.dir/src/core_ghidra.cpp.o] Error 1
164.7 make[1]: *** [CMakeFiles/Makefile2:119: CMakeFiles/core_ghidra.dir/all] Error 2
164.7 make: *** [Makefile:130: all] Error 2
------
```
See full log at https://github.com/rizinorg/rizin/actions/runs/5977713374/job/16221049488

--- a/librz/include/rz_analysis.h
+++ b/librz/include/rz_analysis.h
@@ -731,14 +731,14 @@ typedef enum {
 } RzAnalysisVarOriginKind;
 
 static const char *RzAnalysisVarKind_strings[] = {
+	[RZ_ANALYSIS_VAR_KIND_INVALID] = "invalid",
 	[RZ_ANALYSIS_VAR_KIND_FORMAL_PARAMETER] = "formal_parameter",
 	[RZ_ANALYSIS_VAR_KIND_VARIABLE] = "variable",
-	[RZ_ANALYSIS_VAR_KIND_INVALID] = "invalid",
 };
 
 static const char *RzAnalysisVarOriginKind_strings[] = {
-	[RZ_ANALYSIS_VAR_ORIGIN_DWARF] = "DWARF",
 	[RZ_ANALYSIS_VAR_ORIGIN_NONE] = "none",
+	[RZ_ANALYSIS_VAR_ORIGIN_DWARF] = "DWARF",
 };
 
 #define RZ_ANALYSIS_AS_STRING_IMPL(T, name, strings) \
@@ -751,10 +751,10 @@ static const char *RzAnalysisVarOriginKind_strings[] = {
 	static inline T rz_analysis_##name##_from_string(const char *s) { \
 		for (int i = 0; i < RZ_ARRAY_SIZE(strings); ++i) { \
 			if (RZ_STR_EQ(s, strings[i])) { \
-				return i; \
+				return (T)i; \
 			} \
 		} \
-		return 0; \
+		return (T)0; \
 	}
 
 RZ_ANALYSIS_AS_STRING_IMPL(RzAnalysisVarKind, var_kind, RzAnalysisVarKind_strings);

